core/src/main/java/com/datastax/oss/driver/internal/core/session/DefaultSession.java

```diff
@@ -431,7 +433,7 @@ private void afterInitialSchemaRefresh(CqlIdentifier keyspace) {
       }
     }
 
-    private void notifyLifecycleListeners() {
+    private void notifyListeners() {
       for (LifecycleListener lifecycleListener : context.getLifecycleListeners()) {
         try {
           lifecycleListener.onSessionReady();
```

Comment:
Shouldn't we pass the session here too?

Reply:
This has worked without it so far, so I see no reason to change it now. If we reuse `LifecycleListener` in the future, and the new use case requires the session, we can change it then.